core/block_svg.js

```diff
@@ -1035,13 +1038,12 @@ Blockly.BlockSvg.prototype.disposeUiEffect = function() {
   clone.bBox_ = clone.getBBox();
   // Start the animation.
   clone.startDate_ = new Date();
-  Blockly.BlockSvg.disposeUiStep_(clone, this.RTL);
+  Blockly.BlockSvg.disposeUiStep_.bind(this)(clone, this.RTL);
```

Comment:
Javascript is definitely not my main language, but is this correct syntax? what is the second parenthesis doing?